packages/core/src/components/popover/popover.scss

```diff
@@ -1,6 +1,5 @@
 @use '../../helpers';
 @use '../../mixins';
-@use 'sass:map';
```

Comment:
Would be good for linting to catch this, but I know earlier I was looking into this and did not find a out-of-the-box solution.

Reply:
Yea automate all the things!
I missed this in the other PR, as well as several changes to Popover's stories, could transfer the changes over if you think that's necessary.

Reply:
Moved some Popover changes over, should make this PR more concise and targeted.